### PR TITLE
Camel K 1.4.0 release and blog post

### PR DESCRIPTION
**Pull request by nicolaferraro**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: content/blog/2021/04/camel-k-140-Whatsnew/index.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+Users can bind them to a specific destination by writing a YAML binding file, as explained in the [specific documentation related to each Kamelet](/camel-kamelets/latest/aws-sqs-source.html#_knative_source).
+
+Or, you can use the new `kamel bind` command (see below).
+
+**Note:** it's easy to write your own Kamelet and publish it to the Apache Catalog. Take a look at the [Kamelets developer guide](/camel-k/latest/kamelets/kamelets-dev.html).
+
+## Kamel bind command
+
+We've added a `bind` subcommand to the  `kamel` CLI that provides a new way to use Kamelets directly when you need to connect them to Knative channels, Kafka topics and any other endpoint.
```
Comment (zregvart) [inline content/blog/2021/04/camel-k-140-Whatsnew/index.md:30]:
```suggestion
We've added a `bind` subcommand to the  `kamel` CLI that provides a new way to use Kamelets directly when you need to connect them to Knative channels, Kafka topics, and any other endpoint.
```

File: content/blog/2021/04/camel-k-140-Whatsnew/index.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```
+kamel bind earthquake-source channel:earthquakes
+```
+
+This command creates the KameletBinding resource for you and the Camel K operator does the rest to bring that data into your channel.
+Data is produced using the [Earthquake Source](/camel-kamelets/latest/earthquake-source.html) Kamelet available in the embedded Katalog.
+
+You can use any other Kamelet from the catalog using the `kamel bind` command.
+
+You can also target any other Kubernetes reference that is supported by Camel K, for example, sink into a [Strimzi KafkaTopic](https://strimzi.io/), using a full qualified reference, for example:
```
Comment (zregvart) [inline content/blog/2021/04/camel-k-140-Whatsnew/index.md:43]:
```suggestion
You can also target any other Kubernetes reference that is supported by Camel K, for example, sink into a [Strimzi KafkaTopic](https://strimzi.io/), using a fully qualified reference, for example:
```

File: content/blog/2021/04/camel-k-140-Whatsnew/index.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+```
+kamel bind earthquake-source kafka.strimzi.io/v1beta1:KafkaTopic:mytopic
+```
+
+Of course the command also supports plain Camel URIs, that are useful especially when you're developing a new Kamelet. For example you can write:
```
Comment (zregvart) [inline content/blog/2021/04/camel-k-140-Whatsnew/index.md:49]:
```suggestion
Of course, the command also supports plain Camel URIs, which are useful especially when you're developing a new Kamelet. For example, you can write:
```

File: content/blog/2021/04/camel-k-140-Whatsnew/index.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+The user can now *edit the file to remove sensitive information* (which the command may not be able to tell apart), then 
+share it with developers to have much better insights.
+
+## Stability and compatibility
+
+We focused a lot on stability and improved compatibility with other tools of the ecosystem.
+
+**Knative support** (0.22.0) has been improved by fixing compatibility issues that sometimes caused multiple revisions
+to be present for the same service. We've also changed the way channels and brokers are bound to the integrations, so that now it's possible
+to bind integrations to multiple channels and even create sequences of integrations attached to channels without any issue (e.g. [#2190](https://github.com/apache/camel-k/pull/2190), [#2115](https://github.com/apache/camel-k/pull/2115)).
```
Comment (zregvart) [inline content/blog/2021/04/camel-k-140-Whatsnew/index.md:86]:
```suggestion
to be present for the same service. We've also changed the way channels and brokers are bound to the integrations. Now it's possible
to bind integrations to multiple channels and even create sequences of integrations attached to channels without any issue (e.g. [#2190](https://github.com/apache/camel-k/pull/2190), [#2115](https://github.com/apache/camel-k/pull/2115)).
```

File: content/blog/2021/04/camel-k-140-Whatsnew/index.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+We've improved **installation options**, letting you configure things that may be important in a production environment, like [setting toleration](https://github.com/apache/camel-k/pull/2114) or using a [secured maven repository](https://github.com/apache/camel-k/pull/2180).
+At the same time, we've fixed compatibility with recent dev environments, e.g. letting you [smoothly install Camel K in K3S](/camel-k/latest/installation/registry/k3s.html).
+
+We also kept doing changes to continuously improve speed. In the runtime side of Camel K, we now use the [Quarkus fast-jar format](https://github.com/apache/camel-k/pull/1931) to reduce boot times. In the operator side, it's possible to install Camel K globally in a cluster and have much faster build times by [sharing base images across the cluster](https://github.com/apache/camel-k/pull/2058).
```
Comment (zregvart) [inline content/blog/2021/04/camel-k-140-Whatsnew/index.md:91]:
```suggestion
We also kept doing changes to continuously improve speed. On the runtime side of Camel K, we now use the [Quarkus fast-jar format](https://github.com/apache/camel-k/pull/1931) to reduce boot times. And last but not least on the operator side, it's possible to install Camel K globally in a cluster and have much faster build times by [sharing base images across the cluster](https://github.com/apache/camel-k/pull/2058).
```

**Review by zregvart — COMMENTED** [review]
I've run a spell/grammar check. Some suggestions inline.

**Comment by nicolaferraro** [comment]
Thanks @zregvart

**Comment by zregvart** [comment]
There was an GitHub Actions issue running the checks, I think this is safe to merge though.

**Pull request merged**